docs/kytConfig.md

```diff
@@ -37,6 +37,8 @@ modifyWebpackConfig: (baseConfig, options) => {
 }
 ```
 
+> **NOTE:** [babel-polyfill](https://babeljs.io/docs/usage/polyfill/) is included in the Webpack configs' entry.main arrays by default. If you use `modifyWebpackConfig()` to override the entry properties and wish to keep using the polyfill, the new entries should go after the polyfill.
```

Comment:
Hey @dting I think the problem is because the secret holding the api token for the repo isn't being set since you're PR is from a different repo:

https://docs.travis-ci.com/user/environment-variables/#Defining-Variables-in-Repository-Settings

```Similarly, we do not provide these values to untrusted builds, triggered by pull requests from another repository.```

I'm going to keep looking into this. Can you run the e2e tests locally and let me know if they pass? Thanks!